app/src/main/res/values/strings.xml

```diff
@@ -35,4 +35,6 @@
     <string name="nav_drawer_username">Username</string>
     <string name="nav_drawer_user_description">More Info on user</string>
     <string name="send_message">Send</string>
+    <string name="desc_publication">This is the publication of the user</string>
```

Comment:
According to CI warnings, some strings are unused. You can can remove them as well.